### DISCUSSION TITLE
Unable to install Charting package in PCF Project

### DISCUSSION THREAD

**Discussion by Skubakoob** [microsoft/fluentui#28207] [Q&A]
I've created a PCF project using the newer "ComponentFramework.ReactControl" as per the docs.

When I try to install the charting package using "npm install @fluentui/react-charting", I get this error:

npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: pcf-project@1.0.0
npm ERR! Found: @fluentui/react@8.29.0
npm ERR! node_modules/@fluentui/react
npm ERR!   @fluentui/react@"8.29.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @fluentui/react@"^8.110.2" from @fluentui/react-charting@5.16.41
npm ERR! node_modules/@fluentui/react-charting
npm ERR!   @fluentui/react-charting@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

As I'm using the framework libraries, I know I can't fiddle with the fluent-ui version. Is this an issue with the dependencies within the charting package? Should I raise this as a bug?

**Comment by AtishayMsft** [comment]
@Skubakoob You could either accept the older peer dependency version using the --legacy-peer-deps flag.
Or alternatively install the older version **5.3.28** of react-charting that depends on the same 8.29.0 version of @fluentui/react that your application already uses.
